kinder/OWNERS

```diff
@@ -0,0 +1,9 @@
+# See the OWNERS file documentation:
+#  https://github.com/kubernetes/community/blob/master/contributors/devel/owners.md
+approvers:
+- fabriziopandini
+- neolit123
+reviewers:
```

Comment:
Do you need to add you and @neolit123 as reviewer as well?

Reply:
Good point. I will do this immediately after this one merge